qbft/spectest/tests/messages/msg_non_zero_identifier.go

```diff
@@ -9,12 +9,14 @@ import (
 
 // MsgNonZeroIdentifier tests Message with len(Identifier) == 0
 func MsgNonZeroIdentifier() *tests.MsgSpecTest {
-	msg := testingutils.SignQBFTMsg(testingutils.Testing4SharesSet().Shares[1], types.OperatorID(1), &qbft.Message{
+	ks := testingutils.Testing4SharesSet()
+
```

Comment:
it doesn't test zero length identifier, it's nil here

Reply:
@alonmuroch I left it unchanged, it's `Identifier: []byte{},`, or do you mean anything else?